tests/test_views.py

```diff
@@ -116,6 +116,7 @@ def check_status(self):
         assert response.status_code == 200, response.content.decode('utf-8')
         assert response['content-type'] == 'text/html; charset=utf-8'
         assert b'so so' in response.content, response.content
+        HEALTH_CHECK['WARNINGS_AS_ERRORS'] = True
```

Comment:
this will evaluate only when all `assert` statements will pass.
`pytest` has neat fixture [`monkeypatch`](https://docs.pytest.org/en/6.2.x/monkeypatch.html#monkeypatching-dictionaries) that can be used to override `HEALTH_CHECK` dictionary just for one test. Just use `monkeypatch.setitem` to update `WARNINGS_AS_ERRORS` key:


```python
monkeypatch.setitem(HEALTH_CHECK, 'WARNINGS_AS_ERRORS', False)
````

Reply:
This project also uses `pytest-django` that has great fixture - `settings` that can be also used to modify `django-health-check` settings:

```python
settings.HEALTH_CHECK['WARNINGS_AS_ERRORS'] = True
```

And personally, I think it's the best way to write this test because we are already using `pytest-django`'s `client` fixture.

Reply:
Thanks a lot for your comments! `monkeypatch` is a great solution! I strongly agree that your fix is much better than my original one.

By the way, I tried to use `settings` but it failed in the following way.
```
tests/test_views.py:113: in test_warning
    settings.HEALTH_CHECK['WARNINGS_AS_ERRORS'] = True
/home/sturmian/.local/lib/python3.8/site-packages/pytest_django/fixtures.py:347: in __getattr__
    return getattr(settings, item)
/home/sturmian/.local/lib/python3.8/site-packages/django/conf/__init__.py:83: in __getattr__
    val = getattr(self._wrapped, name)
E   AttributeError: 'Settings' object has no attribute 'HEALTH_CHECK'
```

Please let me know if you would like to improve my current fix, thanks!

Reply:
It seems like `django-health-check` is not initialized in `tests/test_app/settings.py`, so let's stick to `monkeypatch` :+1: